### DISCUSSION TITLE
请问kratos的微服务代码如何调试

### DISCUSSION THREAD

**Discussion by wangshuo5597** [go-kratos/kratos#1868] [Q&A] [answered]
使用vscode，之前用gin框架开发直接F5进入debug模式。但是kratos服务只能支持kratos run 运行，没法进断点了。请教各位如何调试呢?

**Comment by Windfarer** [comment, marked as answer]
可以cd进cmd下面的入口package里run

**Reply by wangshuo5597** [reply]
>

> 可以cd进cmd下面的入口package里run

之前在cmd里找到main.go没运行起来，提示找不到config文件，过段时间才发现kratos的版本过低导致的问题，升级到2.x就没问题了

**Comment by minicoder1024** [comment]
![image](https://user-images.githubusercontent.com/12459182/210969746-98a6c4f1-fef8-40f4-b042-e792257541c1.png)
